src/CycloneDX/Helpers/Helpers.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const toPurl = (name: string, version: string, group = ''): string => {
```

Comment:
hahaha that's all there was to packageurl-js! 

We left-pad'd ourselves! 🤣

Reply:
Yeah, I get why they used it (why not? It's a package that does what they want), but it was simple enough to just implement.